corehq/apps/domain/forms.py

```diff
@@ -169,9 +169,13 @@ class SnapshotSettingsForm(SnapshotSettingsMixin):
         help_text=ugettext_noop("An optional image to show other users your logo or what your app looks like"))
     video = CharField(label=ugettext_noop("Youtube Video"), required=False,
         help_text=ugettext_noop("An optional youtube clip to tell users about your app. Please copy and paste a URL to a youtube video"))
+    documentation_file = forms.FileField(label=ugettext_noop("Documentation File"), required=False,
+        help_text=ugettext_noop("An optional file to tell users more about your app."))
     cda_confirmed = BooleanField(required=False, label=ugettext_noop("Content Distribution Agreement"))
 
     def __init__(self, *args, **kw):
+        self.dom = kw.pop("domain", None)
```

Comment:
Agreed dom is misleading, but it was already being used as dom: https://github.com/dimagi/commcare-hq/pull/5689/files#diff-21bd1c0e52720275d9cf19e3dd556434L1622. Presumably something depends on it being called dom. I think it should be changed but probably in a separate PR